specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2023-02-01/vaults.json

```diff
@@ -1190,6 +1190,44 @@
         }
       }
     },
+    "SoftDeleteSettings": {
+      "description": "Soft delete Settings of vault",
+      "type": "object",
+      "properties": {
+        "softDeleteState": {
+          "enum": [
+            "Invalid",
+            "Enabled",
+            "Disabled",
+            "AlwaysON"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SoftDeleteState",
+            "modelAsString": true
+          }
+        },
+        "softDeleteRetentionPeriodInDays": {
+          "format": "int32",
+          "description": "Soft delete retention period in days",
+          "type": "integer"
+        }
+      }
+    },
+    "MultiUserAuthorization": {
+      "description": "MUA Settings of vault",
+      "enum": [
+        "Invalid",
+        "Enabled",
+        "Disabled"
+      ],
+      "type": "string",
+      "readOnly": true,
```

Comment:
I don't think readOnly means anything in the context of an enum definition?

Reply:
The value for MultiUserAuthorization can't be set and is readonly. Also this is how it was added in later APi verisons - https://github.com/Azure/azure-rest-api-specs/pull/24211/files#diff-23c7e09767e1c7b29e567b0fbe0c06ca6d04f326df34f1e1d27130bb96a2ae61 

should I still remove it ?